scripts/uiupdate.sh

```diff
@@ -5,18 +5,15 @@ set -e
 if [ -z "$UI_VERSION_FILE" ]; then
 	echo "Must set UI_VERSION_FILE"; exit 1
 fi
-if [ -z "$UI_CLONE_DIR" ]; then
-	echo "Must set UI_CLONE_DIR"; exit 1
-fi
 
 shafileabs="$(pwd)/${UI_VERSION_FILE}"
-cd "${UI_CLONE_DIR}"
-V="$(git log -n1 --pretty=oneline)"
-echo "==> Setting UI version to: $V"
+V="$(git ls-remote https://github.com/hashicorp/boundary-ui main)"
```

Comment:
If we remove UI_COMMITISH we also need to update the release process: https://hashicorp.atlassian.net/wiki/spaces/ICU/pages/1459880389/Binary+and+Docker+Release#UI-Version

Reply:
Sorry, I was only looking at this from the scope of `uiupdate.sh`, not who `update-ui-version` would really be using it. It now uses `$UI_COMMITSH` instead of just main. I added a check for when a commit is provided via `$UI_COMMITISH`. It will check the response from `git ls-remote` and set `$V` to `$UI_COMMITISH` if the test is true.